pdp/storage.go

```diff
@@ -143,15 +143,17 @@ func (c *command) describe() string {
 		}
 	}
 
+	opPath := strings.Join(qpath, "/")
 	if nil != c.entity {
 		if evaluable, ok := c.entity.(Evaluable); ok {
```

Comment:
Why we don't case for rules here? Evaluable is an interface only policy and policy set while for rule message remains as before. We could say for example that we are adding permit or deny rule.